### PR TITLE
Update and rename test_commit_1 to test_commit_2

### PR DESCRIPTION
**Pull request by hasalfernando**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by main-competitor** [comment]
This looks good really!

**Review by main-competitor — COMMENTED** [review]
Adding the review comment 1. Maybe this is a change.

**Review by main-competitor — APPROVED** [review]
This looks good to me. Comment 2.

**Comment by hasalfernando** [comment]
Thanks

File: test_commit_2
Comment (main-competitor) [inline test_commit_2:1]: This seems to be nice

**Review by main-competitor — CHANGES_REQUESTED** [review]

File: test_commit_2
Comment (main-competitor) [reply test_commit_2:1]: But need to clear

Comment (hasalfernando) [reply test_commit_2:1]: Trying to test the 3rd comment on this PR. Let's see.

Comment (main-competitor) [inline test_commit_2:1]: Yeah looks good can merge.

**Review by main-competitor — APPROVED** [review]

**Pull request merged**